docs/prometheus-exporter.md

```diff
@@ -150,3 +150,33 @@ Always has the value 1.
 
 Timestamp of starting up the node (Unix time in milliseconds).
 
+### `grpc_request_duration_seconds`
+
+Histogram tracking the total number of gRPC requests received and their duration. Labelled with the gRPC method name (`method=<name>`) and the gRPC response status (`status=<status>`).
+
+The duration is tracked for the underlaying HTTP2 request response time, meaning the streaming gRPC methods which could span over multiple HTTP2 requests are counted for each.
```

Comment:
It's hard to understand what you mean "for each" at the end. Can you elaborate please.

Reply:
I have updated the documentation since it did not reflect what actually happens